src/App.tsx

```diff
@@ -1,10 +1,44 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import { Header } from './components/Header/Header';
+import { ToDoList } from './components/ToDoList/ToDoList';
+import { Error } from './components/Error/Error';
+import { Footer } from './components/Footer/Footer';
```

Comment:
TIP: use the index.tsx in ./components and re-export every component so that the import could be simplified to:
`import { Header, ToDolist, Error, Footer } from "./components"`